docs/projects/tcet-linux-website/frontend-components/footer.mdx

```diff
@@ -0,0 +1,136 @@
+
+### Copyright and Email Component
+
+This Component created by our team, consists of Copyright text of the Organisation.
+
+  ```jsx title="index.astro" {3-4} showLineNumbers
+   //Top Section code
+   <div class=" w-full  flex flex-col-reverse sm:flex-row   sm:justify-between items-center sm:gap-0 gap-4">
+    <div class="  text-xs  text-[#D0D5DD] ">Copyright 2022-2023 @ TCET Opensource.</div>
+    <a class="  text-xs  text-[#D0D5DD]" href="mailto:linux@tcetmumbai.in">linux@tcetmumbai.in</a> 
+  ```
+  1. We created the First `<div>` as the Sectional div for Bottom Section.
+
+  2. Second `<div>` is created to display the Copyright text of this componenet.
+
+  3. We created the `<a>` tag in-order to display Email link in **href** for LINUX-TCET Mumbai. 
+
+:::info
+The Copyright Text may be converted into accessible text with link to the Copyright page of the Organisation.
+:::
+
+### Social Media Component 
+
+This Component contains various social media links of TCET Open Source Organisation.
+
+ ```jsx title="index.astro" {3-4} showLineNumbers
+   //Copyright and Email Component code
+    <div class="flex text-[#D0D5DD] gap-8">
+      <a href="https://www.linkedin.com/company/tcet-opensource/" target="_blank"><img src="/Footer/linkdin.png" alt="" class="h-5 cursor-pointer" /></a>
+      <a href="https://www.youtube.com/@TCETMumbaiOfficial" target="_blank"> <img src="/Footer/Youtube.png" alt="" class="h-5 cursor-pointer" /></a>
+      <a href="https://twitter.com/tcetopensource" target="_blank"> <img src="/Footer/Twitter.png" alt="" class="h-5 cursor-pointer" /></a>
+      <a href="https://www.instagram.com/tcetopensource/" target="_blank"><img src="/Footer/Instagram.png" alt="" class="h-5 cursor-pointer" /></a>
+      <a href="https://github.com/tcet-opensource/" target="_blank"><img src="/Footer/Github.png" alt="" class="h-5 cursor-pointer" /></a>
+    </div>
+ ```
+1. We have used Basic HTML syntax in this component to fulfill our requirements .
+
+2. The `<a>` tag is used by us to display LinkedIn Platform link in **href** of TCET OpenSource Organisation .
+
+3. Similarly using same syntax, we have linked different Social Media Platform Account of TCET OpenSource Organisation for contact process of user.
+
+:::tip
+Using similar Basic HTML syntax, more **Social Media** links can be added in the Bottom Component of Footer Element of the Website. 
+:::
+
+**After adding all components, the Footer Element can be scene like this :**
+
+import footer from "/docs/projects/tcet-linux-website/assets/footer.jpg";
+
+
+<figure>
+    <img src={footer} style={{ border: "2px solid gray"}} />
+    <center><b><figcaption>Footer Section</figcaption></b></center>
+</figure>
+
+After successfully implementing the Footer component of the [**TCET Linux Website**](https://linux.tcetmumbai.in/) , the next step is to move forward with the implementation of the Copy to Clipboard component.
```

Comment:
Link footer and copy to clipboard component with current and next pages respectively.